core/sdk-core/src/main/java/software/amazon/awssdk/core/SdkServiceClientConfiguration.java

```diff
@@ -53,6 +57,16 @@ public Optional<URI> endpointOverride() {
         return Optional.ofNullable(this.endpointOverride);
     }
 
+    /**
+     *
+     * @return The configured endpoint provider of the SdkClient. If the endpoint provider was not configured, an empty
```

Comment:
If it is not configured by users, wouldn't we return the default one created by the SDK?

Reply:
updated